gammapy/spectrum/dataset.py

```diff
@@ -805,6 +813,12 @@ def stack_edisp(self):
         irf_stacker.stack_edisp()
         self.stacked_edisp = irf_stacker.stacked_edisp
 
+    def stack_gti(self):
+        """Stack GTI
+        """
+        # TODO: use what Regis is implemented for GTI union
+        self.stack_gti = None
```

Comment:
you should have typed `self.stacked_gti = None`.
Anyway I would `raise NotImplemented`